src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/integration/service/PrisonOffenderEventListenerIntTest.kt

```diff
@@ -92,7 +92,7 @@ internal class PrisonOffenderEventListenerIntTest : SqsIntegrationTestBase() {
   @Test
   fun `prisoner with MERGE numbers is processed`(): Unit = runBlocking {
     // Given
-    val bookingId = 1294134L
+    val bookingId = 1294133L
```

Comment:
Despite all integration tests having `repository.deleteAll()` in `BeforeEach` and `AfterEach` function this test was still having issues, so changed the `bookingId` to avoid it